src/core/Akka/Dispatch/ChannelSchedulerExtension.cs

```diff
@@ -60,7 +60,7 @@ public static ChannelTaskScheduler Get(ActorSystem system)
         public ChannelTaskScheduler(ExtendedActorSystem system)
         {
             //config channel-scheduler
-            var config = system.Settings.Config.GetConfig("akka.channel-scheduler");
+            var config = system.Settings.Config.GetConfig("akka.actor.channel-scheduler");
```

Comment:
LGTM

Reply:
Is this correct? @Zetanova said the `channel-scheduler` is a standalone feature, does not belong to the actor config

Reply:
He said `channel-scheduler` inside the actor config was a mistake.

Reply:
The `channel-scheduler` is not directly related with actor or dispatchers
the `channel-executor` is using the `channel-scheduler` to queue work-items

Reply:
If we move it into `akka` - is that going to break current users on 1.4.32? Or since the config reading is already broken, should that not be an issue @Arkatufus ?

Reply:
It shouldn't be an issue since it doesn't work in the first place.

Reply:
If it doesn't have anything to do with actors and dispatchers, why is it in the Dispatch namespace?

Reply:
> * If it doesn't have anything to do with actors and dispatchers, why is it in the Dispatch namespace?
The config was always planed to be under `akka.channel-scheduler`, I made simply a copy-past mistake 

> * Is there anything else in Akka that is using this beside actors?
The `ChannelTaskScheduler` is a singleton and currently only used by the new `channel-executor` dispatcher
It is thread safe and could be used in any place as a regular `TaskScheduler`

> * Maybe it shouldn't be a separate component?
It need to be a singleton

> * Is using the component pattern an anti-pattern? Is it over-engineering it a bit?
It need to be a singleton and it can be lazy initialized

> * Will putting it outside the `akka.actor` sub config makes it harder to explain this to users?
The `ChannelTaskScheduler` is a standalone component. 

>   * Because the name is ambiguous, what is "channel scheduler"?
Because the `channel-executor` was already named, I decided to skip on the `task` and use `channel`
the correct naming would be `channel-task-scheduler` or `priority-task-scheduler`

>   * Does it have anything to do with distributed pub sub? cluster?
the system and remote dispatcher has priority "high" as default to always priorities cluster heartbeats and the rest over regular tasks.

>   * What is a channel?
The word `channel` comes from the namespace used `System.Threading.Channels`

>   * Why do I have to configure something I want to use inside an actor so far away from the actor configuration itself?
The default config is just working fine, but of course should be explicitly configurable. 
I just copy-pasted the default block into the wrong section on my PR